examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -55,49 +55,29 @@ async def _main():
         try:
             client = grpc_nidaqmx.NiDAQmxStub(channel)
 
-            async def raise_if_error(response):
-                """Raise an exception if an error was returned."""
-                if response.status:
-                    response = await client.GetErrorString(
-                        nidaqmx_types.GetErrorStringRequest(error_code=response.status)
-                    )
-                    raise Exception(f"Error: {response.error_string}")
-
-            async def raise_if_error_async(awaitable_call):
-                """Await response, then raise an exception if an error was returned."""
-                response = await awaitable_call
-                await raise_if_error(response)
-                return response
-
-            create_response: nidaqmx_types.CreateTaskResponse = await raise_if_error_async(
-                client.CreateTask(nidaqmx_types.CreateTaskRequest())
-            )
+            nidaqmx_types.CreateTaskResponse = client.CreateTask(nidaqmx_types.CreateTaskRequest())
             task = create_response.task
 
-            await raise_if_error_async(
-                client.CreateAIVoltageChan(
-                    nidaqmx_types.CreateAIVoltageChanRequest(
-                        task=task,
-                        physical_channel=PHYSICAL_CHANNEL,
-                        min_val=-10.0,
-                        max_val=10.0,
-                        terminal_config=nidaqmx_types.InputTermCfgWithDefault.INPUT_TERM_CFG_WITH_DEFAULT_CFG_DEFAULT,
-                        units=nidaqmx_types.VoltageUnits2.VOLTAGE_UNITS2_VOLTS,
-                    )
+            client.CreateAIVoltageChan(
```

Comment:
I haven't used Python asyncio but it looks like this creates a bunch of coroutines without actually executing them. I suspect that each one of this calls needs an `await`.

Reply:
Added `await`s everywhere that we were previously calling `raise_if_error_async()`.